### PR TITLE
fix: attribute value for compose.yaml `name`

### PR DESCRIPTION
**Pull request by cpate4**

### CODE REVIEW

File: scripts/compose.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: ${{values.projecSlug}}
+name: ${{values.projectSlug}}
```
Comment (mikeflinn) [inline scripts/compose.yaml:1]:
```suggestion
name: ${{values.name}}
```

Thanks for catching this, if we want to pattern match on what is in the upstream this should probably be `name`

**Review by mikeflinn — APPROVED** [review]

File: scripts/compose.yaml (hunk 1 above)
Comment (cpate4) [reply scripts/compose.yaml:1]: I think we had it right based on this file ...

[first line of yaml](https://github.com/DataBiosphere/terra-java-project-template/blob/ac5d9c8e8469640dcaa28730dabcd18be06b1933/scripts/compose.yaml#L1) here:
```yaml
name: ${{values.projectSlug}}
...
```

and [6th line of compose.yaml](https://github.com/DataBiosphere/terra-java-project-template/blob/ac5d9c8e8469640dcaa28730dabcd18be06b1933/scripts/compose.yaml#L6) here
```yaml
...
services:
  service: 
    image: ${{values.name}}
...
```

Comment (cpate4) [reply scripts/compose.yaml:1]: ... I'll wait for confirmation before doing anything - 🙌

Comment (mikeflinn) [reply scripts/compose.yaml:1]: Ah yeah, you are right

**Pull request merged**